app/helpers/orchid/display_helper.rb

```diff
@@ -36,6 +36,7 @@ def metadata_create_field_link(api_field, item)
     item_label = value_label(api_field, item)
     link_to item_label, prefix_path(route_path, search_params),
       rel: "nofollow"
+    "<dd>#{link}</dd>"
```

Comment:
There's no `link` variable in scope here. The return of `link_to` would have to be assigned to one first. But I think there's another oversight here. This is the only place we have a `<dd>` and `</dd>` tags. This code here only affects output when the `metadata` method isn't called with `link: false`, but it is in many places. So they won't be wrapping non-link metadata info as the non-centralized `<li>` tags in the template were before. I think we actually want the `<dd>` tags in the `.map` above.

Reply:
Fixed as suggested